recipes/scikit-fda/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "scikit-fda" %}
+{% set version = "0.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/scikit-fda-{{ version }}.tar.gz
+  sha256: 69f683b866813d61e209f2213ebbe266b3762588569f39a9b0065b950ba260f2
+
+build:
+  skip: true  # [py<38]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - cython
```

Comment:
I see Cython listed in the requirements, but it doesn't seem to be used anywhere?
```suggestion
```